src/themes/teams-high-contrast/siteVariables.ts

```diff
@@ -1,10 +1,17 @@
-import { white } from '../teams/siteVariables'
+import { white, green } from '../teams/siteVariables'
 
 //
 // COLORS
 //
 export const black = '#000'
 
+export const hyperlinkColor = '#ffff01'
+export const disabledColor = '#3ff23f'
```

Comment:
This feels too generic to be useful in all color contexts.  How can we have a single color for "disabled" without knowing if it is for a foreground, background, border, etc or which component it is for?

Reply:
I can rename to _accessibleYellow_ and _accessibleGreen_ for now. Although, maybe it should just be a general green and yellow option, like _yellow0x_ and _green0x_...